fields.go

```diff
@@ -544,3 +536,15 @@ func consumeTagOption(in string) (string, int, error) {
 func isLetterOrDigit(r rune) bool {
 	return r == '_' || unicode.IsLetter(r) || unicode.IsNumber(r)
 }
+
+// boolsCompare compares x and y, ordering false before true.
```

Comment:
Is it worth keeping the reference to https://go.dev/issue/61643 here so we don't forget that there is a declined proposal for this?

Reply:
The challenge is there's been so many bool-to-int or compare-bools proposals. 61643 isn't exactly unique. I think a standalone helper that is equivalent to bools.Compare will be useful for static analysis to one-day prove how often this is used rather than a reference to the issue.

Reply:
It's okay, I'm sure the next bool-to-int proposal will get approved :wink: